src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -527,3 +545,18 @@ void MainPage::OnNavItemInvoked(MUXC::NavigationView ^ /*sender*/, _In_ MUXC::Na
 {
     NavView->IsPaneOpen = false;
 }
+
+void MainPage::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    Model->AlwaysOnTopButtonClick(sender, e, 0, 0);
```

Comment:
You might consider having the `width` and `height` params be optional with 0 as the default value, simplifying the call.

Reply:
WinRT does not allow optional parameters for public functions. See [here](https://social.msdn.microsoft.com/Forums/en-US/4800e079-fb2e-4488-a76d-64b3a04bda76/optional-parameters-for-winrt-component)

Reply:
I'm always surprised by this limitation.

But you can write:
`void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
{
    HandleAlwaysOnTopButtonClick(sender, e, 0, 0);
}`

or 

`void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
{
    AlwaysOnTopButtonClick(sender, e, 0, 0);
}`